be/src/olap/aggregate_func.h

```diff
@@ -90,33 +86,98 @@ class AggregateInfo {
     FieldAggregationMethod _agg_method;
 };
 
+template<FieldType field_type>
 struct BaseAggregateFuncs {
-    // Default init function will set to null
-    static void init(char* dst, Arena* arena) {
-        *reinterpret_cast<bool*>(dst) = true;
+    static void init(RowCursorCell* dst, const char* src, bool src_null, Arena* arena) {
+        dst->set_is_null(src_null);
+        if (src_null) {
+            return;
+        }
+
+        typedef typename FieldTypeTraits<field_type>::CppType CppType;
```

Comment:
Can we use types. copy_with_arena to reuse some code, For example AggregateFuncTraits<OLAP_FIELD_AGGREGATION_NONE, OLAP_FIELD_TYPE_VARCHAR>

Reply:
OK